v2/inc/c_logging/log_sink.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef LOG_SINK_H
+#define LOG_SINK_H
+
+#include "c_logging/log_context.h"
+
+typedef void (*LOG_SINK_LOG_FUNC)(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, char* message, const char* file, const char* func, int line);
+
+typedef struct LOG_SINK_TAG
+{
+    LOG_SINK_LOG_FUNC log_sink_log;
+} LOG_SINK;
```

Comment:
>LOG_SINK [](http://example.com/codeflow?start=2&length=8)

Is this a wanna-be interface? [with just 1 function?]

Reply:
line 3 of the .md says it is. Then consider naming it with `_IF`

Reply:
Renamed to _IF. As part of that also extracted LOG_LEVEL in its own header.